dotCMS/src/main/java/com/dotcms/ai/api/BulkEmbeddingsRunner.java

```diff
@@ -37,8 +37,8 @@ public void run() {
                 List<Field> fields = contentlet.getContentType().fields().stream().filter(f -> embeddingsForm.fieldsAsList().contains(f.variable().toLowerCase())).collect(Collectors.toList());
 
                 // if a velocity template is passed in, use it.  Otherwise, try the fields
-                if (!EmbeddingsAPI.impl().generateEmbeddingsforContent(contentlet, embeddingsForm.velocityTemplate, embeddingsForm.indexName)) {
-                    EmbeddingsAPI.impl().generateEmbeddingsforContent(contentlet, fields, embeddingsForm.indexName);
+                if (!EmbeddingsAPI.impl().generateEmbeddingsForContent(contentlet, embeddingsForm.velocityTemplate, embeddingsForm.indexName)) {
+                    EmbeddingsAPI.impl().generateEmbeddingsForContent(contentlet, fields, embeddingsForm.indexName);
```

Comment:
Let's make sure the runners call API methods only and that we have the proper transaction annotations for these API methods.

Reply:
Will be done by @vicozizou in the next PR